docs/framework/wcf/samples/using-the-wcf-moniker-with-com-clients.md

```diff
@@ -73,7 +73,7 @@ gacutil.exe /i client.dll
   
  ComCalcClient.vbs client application uses the `GetObject` function to construct a proxy for the service, using the service moniker syntax to specify the address, binding, and contract for the service.  
   
-```  
+```vb
```

Comment:
This isn't Visual Basic, its VBScript. I don't think a language identifier for VBScript is supported. There shouldn't be a language identifier here. This is also true of lines 92, 104,137, 149, and 172.

Reply:
VBScript is a supported highlight language

Reply:
Is its language identifier `vbscript`, @thraka?